### PR TITLE
tests: actualize skips

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: s3tests_boto3/functional/test_s3.py [hunk 1]
```diff
@@ -8838,83 +8838,44 @@ def trigger(self):
...
-    upload_id = response["UploadId"]
-
-    fp_a = FakeWriteFile(file_size, "A", lambda: action.trigger())
-
-    response = client.upload_part(
-        UploadId=upload_id, Bucket=bucket_name, Key=key_name, PartNumber=1, Body=fp_a
-    )
-
     parts.append({"ETag": response["ETag"].strip('"'), "PartNumber": 1})
+
```
Comment (roman-khimov) [inline s3tests_boto3/functional/test_s3.py:8870]: Useless whitespace change.

File: s3tests_boto3/functional/test_s3.py [hunk 2]
```diff
@@ -8838,83 +8838,44 @@ def trigger(self):
             self.result = self.action()
 
 
-@pytest.mark.skip(reason="Potential Bug")
-def test_multipart_resend_first_finishes_last():
+@pytest.mark.skip(reason="https://github.com/nspcc-dev/neofs-s3-gw/issues/901")
+def test_multipart_resend():
```
Comment (roman-khimov) [inline s3tests_boto3/functional/test_s3.py:8842]: This change (whole commit) deserves better explanation in the commit message.

File: s3tests_boto3/functional/test_s3.py (hunk 1 above)
Comment (evgeniiz321) [reply s3tests_boto3/functional/test_s3.py]: removed

File: s3tests_boto3/functional/test_s3.py (hunk 2 above)
Comment (evgeniiz321) [reply s3tests_boto3/functional/test_s3.py:8842]: added additional description to the commit

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**